packages/graphql_server/src/graphql/schema.ts

```diff
@@ -2,11 +2,35 @@
 const schema = `
   type Query {
     helloWorld: String!
+    allProjects: [Project!]!
```

Comment:
Please implement the Cursor specification for any lists!!!
https://relay.dev/graphql/connections.htm 
https://relay.dev/docs/guides/graphql-server-specification/#connections
https://graphql.org/learn/pagination/#complete-connection-model

If you want to do this in a new PR, let me know